scripts/validation/check_benchmark_result_provenance.py

```
@@ -0,0 +1,64 @@
+"""Fail-closed provenance manifest checker for benchmark result runs.
```

Pylint (4.0.6)

[refactor] Useless option value for '--disable', 'bad-continuation' was removed from pylint, see https://github.com/pylint-dev/pylint/pull/3571.

(R0022)